runtime/compiler/aarch64/codegen/J9TreeEvaluator.cpp

```diff
@@ -39,3 +43,37 @@ J9::ARM64::TreeEvaluator::generateFillInDataBlockSequenceForUnresolvedField(TR::
    TR_ASSERT_FATAL(false, "This helper implements platform specific code for Fieldwatch, which is currently not supported on ARM64 platforms.\n");
    }
 	
+TR::Register *
+J9::ARM64::TreeEvaluator::flushEvaluator(TR::Node *node, TR::CodeGenerator *cg)
+   {
+   TR::ILOpCodes op = node->getOpCodeValue();
+
+   if (op == TR::allocationFence)
+      {
+      if (!node->canOmitSync())
+         {
+         // Instruction synchronization barrier
+         generateInstruction(cg, TR::InstOpCode::isb, node);
```

Comment:
I don't think this should be an instruction flush.  I think this can be a `DSB 0xF`.